scalableminds/webknossos
# Add button to save current view configuration as default in view mode

### URL of deployed dev instance (used for testing):
- https://updatedefaultlayerconfiginviewmode.webknossos.xyz

### Steps to test:
- Open a dataset in view mode (as admin or dataset manager)
- Change the view mode configuration in a recognizable way (e.g. set histogram min = 100, max = 200)
- Use the new button at the bottom of the layer configuration tab to save the new configuration
- A modal should open up for confirmation. If you cancel the action, no update should be performed.
- If you confirm the modal, a toast should indicate that the updated succeeded
- Use the crog-icon next to the dataset name at the top of the info tab to open the dataset settings
- Go to the view configurations and confirm that the update worked successfully.


## TODOs:
- [x] Test whether this pr works well together with the new layer ordering pr during cover mode for color layers. #7188


### Issues:
- fixes #5183

------
(Please delete unneeded items, merge only when none are left open)
